docs/datastores.md

```diff
@@ -51,6 +51,26 @@ Uses [badger](https://github.com/dgraph-io/badger) as a key value store.
 }
 ```
 
+## badger2ds
+
+Uses [badger2](https://github.com/dgraph-io/badger) as a key value store.
+
+* `syncWrites`: Flush every write to disk before continuing. Setting this to false is safe as go-ipfs will automatically flush writes to disk before and after performing critical operations like pinning. However, you can set this to true to be extra-safe (at the cost of a 2-3x slowdown when adding files).
+* `truncate`: Truncate the DB if a partially written sector is found (defaults to true). There is no good reason to set this to false unless you want to manually recover partially written (and unpinned) blocks if go-ipfs crashes half-way through a adding a file.
+* `compression`: Configure compression. When compression is enabled, every block is compressed using the specified algorithm. This option doesn't affect existing tables. Only the newly created tables are compressed. Compression can be configured to use the snappy algorithm or ZSTD with level 1, 2 or 3.  An unspecified value uses the default configuration.
+* `blockCacheSize`: Specifies how much data cache should hold in memory. If compression is disabled, adding a cache leads to unnecessary overhead which may affect read performance. A value of 0 means no block cache, and an unspecified value uses the default configuration.
+
+```json
+{
```

Comment:
It looks like both badger and badger2 documentation do not specify that you can set `vlogFileSize`.  @Stebalien this is just an oversight right?

Reply:
Yes.

Reply:
@gammazero could you add in the `vlogFileSize` documentation as well?

Reply:
Done.